README.md

```diff
@@ -78,10 +78,10 @@ Instructions
   
   I have tested this with the ebootMOD executable last modified on March 31st, 2011, but other versions may work as well.
 
-* Acquire makeself and unself (Linux and OS X only)
+* Acquire self_rebuilder and unself (Linux and OS X only)
 
-  The failOverflow tools to de- and encrypt are found [here](https://github.com/eltair/fail0verflow). Create a folder called "eboot_tools"
-  in the path of the ToVPatcher.exe binary and copy the makeself and unself binaries there.
+  The failOverflow tools to de- and encrypt are found [here](https://github.com/Evilnat/ps3tools-master_3.XX-4.XX). Create a folder called "eboot_tools"
+  in the path of the ToVPatcher.exe binary and copy the self\_rebuilder and unself binaries there.
```

Comment:
Typo in self_rebuilder here.

Reply:
Actually, it's not a typo but an escape sequence. Otherwise Markdown will recognize the text as italics.

See: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#emphasis

Reply:
Aaah, I see. I'm surprised Markdown parses that as italics in the middle of a word.

Reply:
It was actually my editor (Emacs) that displayed the text as italics starting from "_tools" to "self_. GitHub's Markdown parsing is a little bit different and doesn't recognize it as italics.

So I will pick your preference. Feel free to choose what you prefer and I will rebase it.